realm/realm-library/src/androidTest/java/io/realm/CollectionTests.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import io.realm.entities.AllJavaTypes;
+import io.realm.entities.CyclicType;
+import io.realm.entities.NonLatinFieldNames;
+import io.realm.entities.NullTypes;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Super class for all RealmCollection related tests.
+ * This class only contains configuration and helper methods.
+ */
+public abstract class CollectionTests {
+
+    protected final static long YEAR_MILLIS = TimeUnit.DAYS.toMillis(365);
+
+    // Enumerate all known collection classes from the Realm API.
+    protected enum CollectionClass {
+        MANAGED_REALMLIST, UNMANAGED_REALMLIST, REALMRESULTS;
```

Comment:
There is a quick fix to add them to your dictionary. I would much prefer `REALMLIST` instead of `REALM_LIST`